packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -487,7 +488,7 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
     }
 
     if (this._isElementInput(element)) {
-      if (!this._shouldInputLoseFocus(element as HTMLInputElement, isForward)) {
+      if (!this._tabPressActive && !this._shouldInputLoseFocus(element as HTMLInputElement, isForward)) {
```

Comment:
Consider moving this check into shouldInputLoseFocus since it's directly related to if the input should lose focus when tabbing